src/fontra/views/editor/editor.js

```diff
@@ -1168,16 +1168,19 @@ export class EditorController {
 
     this.basicContextMenuItems.push(MenuItemDivider);
 
-    for (const selectNone of [false, true]) {
-      this.basicContextMenuItems.push({
-        title: selectNone
-          ? translate("action.select-none")
-          : translate("action.select-all"),
-        enabled: () => this.canSelectAllNone(selectNone),
-        callback: () => this.doSelectAllNone(selectNone),
-        shortCut: { keysOrCodes: "a", metaKey: true, shiftKey: selectNone },
-      });
-    }
+    this.basicContextMenuItems.push({
+      title: translate("action.select-all"),
+      enabled: () => this.sceneSettings.selectedGlyph?.isEditing,
+      callback: () => this.doSelectAllNone(false),
+      shortCut: { keysOrCodes: "a", metaKey: true },
```

Comment:
This should have contained `shiftKey: false`. I fixed it here https://github.com/googlefonts/fontra/commit/59e4b030e0346e6f51b7cedda050bb1d17ddd7ec.